### ISSUE TITLE
Navigation bar doesn't expand the entire length for Tablets

### ISSUE THREAD

**Issue by angrezichatterbox** [scribe-org/Scribe-Android#278]
### Terms

- [X] I have searched all [open bug reports](https://github.com/scribe-org/Scribe-Android/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [X] I agree to follow Scribe-Android's [Code of Conduct](https://github.com/scribe-org/Scribe-Android/blob/main/.github/CODE_OF_CONDUCT.md)

### Behavior

Currently the navigation bar does not expand the entire length of the screen in landscape or portrait mode for a tablet. This makes the pages have empty black space at the start and end of the navigation bar as shown in the images given below.

| <img src="https://github.com/user-attachments/assets/41f48e3e-01b9-43e8-bca2-75c4e6f063dd" width="400"/> | <img src="https://github.com/user-attachments/assets/c75b3083-b109-49b2-b2c2-9c5bbefb1d68" width="400"/> |
|--------------------------------------------|--------------------------------------------|

### Device type

Pixel Tablet API 35

**Comment by mnzoidi** [comment]
Hello, I'd like to take on this one. Could you assign this to me?

**Comment by andrewtavis** [comment]
Sure thing, @mnzoidi! Looking forward to working with you on this :)

**Comment by andrewtavis** [comment]
Please let us know if you need any support!

**Comment by mnzoidi** [comment]
The issue seems to be fixed

**Comment by angrezichatterbox** [comment]
> The issue seems to be fixed

Yes, the issue has been fixed by #281. I apologize if you had already begun working on resolving it. Would you like to review some of the other issues? :)

**Comment by angrezichatterbox** [comment]
Closed by #281

**Comment by mnzoidi** [comment]
I'll be sure to check!
